test/scripts/verify-manifests.sh

```
@@ -0,0 +1,183 @@
+echo "========================================"
+echo "Build Verification Summary"
+echo "========================================"
+echo ""
+
+if [ ${#VALIDATED_CHARTS[@]} -gt 0 ]; then
+    echo -e "${BLUE}Helm Charts:${NC}"
+    for chart_dir in charts/*/; do
+        if [ -f "${chart_dir}Chart.yaml" ]; then
+            chart_name=$(basename "$chart_dir")
+            version=$(grep '^version:' "${chart_dir}Chart.yaml" | awk '{print $2}')
+            if [[ " ${VALIDATED_CHARTS[@]} " =~ " ${chart_name} " ]]; then
```

Shellcheck (0.11.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)